_examples/apache-zeppelin-example.adoc

```diff
@@ -0,0 +1,81 @@
+= Apache Zeppelin  Example
+:page-link: apache-zeppelin-example
+:page-weight: 100
+:page-labels: [Notebook, Zeppelin, Spark, R, Pyspark, Scala, Java]
+:page-layout: application
+:page-menu_template: menu_tutorial_application.html
+:page-description: This is an example of how to use Apache Zeppelin
+:page-project_links: ["https://github.com/rimolive/zeppelin-openshift", "https://github.com/rimolive/zeppelin-notebooks"]
+
+[[introduction]]
+== Introduction
+
+Apache Zeppelin is an alternative to project http://jupyter.org/[Jupyter] to provide 
+a notebook to run analysis over a data and plot graphs in order to show to end users
+an orverview about the data.
+
+This example shows how to use the Zeppelin openShift template created to run on top of
+OpenShift, as well as provide 
+
+[[architecture]]
+== Architecture
+
+The central piece is the Apache Zeppelin, used to create notebooks. From zeppelin we will
+connect to the Spark cluster and run some analysis.
+
+[[installation]]
+== Installation
+
+Installing this notebook is straightforward; you simply need to
+create an OpenShift project, deploy a Spark cluster in that project, and
+install and run the notebook image.
+
+=== Prerequisites
+
+Follow the link:/get-started[Get Started] instructions
+to create an OpenShift project with the Oshinko web interface running.
+
+=== Setting up a cluster
+
+From the OpenShift developer console, visit the Oshinko web interface. Use the
+interface to create a new cluster, and take note of what you've called this
+cluster.
+
+=== Setting up the Zeppelin template
+
+In the same project, add the zeppelin template with the following command:
+
+....
+    $ oc create -f https://raw.githubusercontent.com/rimolive/zeppelin-openshift/master/zeppelin-openshift.yaml
+....
+
+After adding the template, create the zeppelin application with the following command:
+
+....
+    $ oc new-app apache-zeppelin --template=apache-zeppelin-openshift --param=APPLICATION_NAME=apache-zeppelin --param=GIT_UTI=https://github.com/rimolive/zeppelin-notebooks.git --param=ZEPPELIN_INTERPRETERS=md
```

Comment:
s/GIT_UTI/GIT_URI/

we should probably bring the zeppelin-notebooks repo into the radanalyticsio org

also, please remove the `$` from the code sample, it will make copying and pasting easier, and i like to format these long commands using the `\` for line extension. so, something like:
```
oc new-app apache-zeppelin --template=apache-zeppelin-openshift \
  --param=APPLICATION_NAME=apache-zeppelin \
  --param=GIT_UTI=https://github.com/rimolive/zeppelin-notebooks.git \
  --param=ZEPPELIN_INTERPRETERS=md
```